composer.json

```diff
@@ -17,17 +17,18 @@
     ],
     "homepage": "https://github.com/php-http/throttle-plugin",
     "require": {
-        "php": ">=7.4",
-        "php-http/client-common": ">=1.3",
-        "symfony/rate-limiter": ">=5.2"
+        "php": "^7.4 || ^8.0",
+        "php-http/client-common": "^1.3 || ^2.0",
+        "symfony/rate-limiter": "^5.2 || ^6.0"
     },
     "require-dev": {
         "nyholm/psr7": "^1.0",
-        "php-http/mock-client": "*",
+        "php-http/mock-client": "^1.0",
         "ramsey/coding-standard": "^2.0",
-        "symfony/phpunit-bridge": ">=6.0"
+        "phpcsstandards/phpcsextra": "^1.0@alpha",
+        "phpcsstandards/phpcsutils": "^1.0@alpha",
```

Comment:
pitty that the ramsey lib depends on not-yet-stable things... i prefer this over the minimum-stability instruction. wdyt @ostrolucky ?

Reply:
it's just cs and require-dev, I don't care that it doesn't use stable things